validator/src/commands/run/execute.rs

```diff
@@ -962,12 +963,15 @@ pub fn execute(
         ..Default::default()
     };
 
-    // Vote shares TPU forward's characteristics, except that we accept 1 connection
-    // per peer and no unstaked connections are accepted.
-    let mut vote_quic_server_config = tpu_fwd_quic_server_config.clone();
-    vote_quic_server_config.max_connections_per_peer = 1;
-    vote_quic_server_config.max_unstaked_connections = 0;
-    vote_quic_server_config.num_threads = tpu_vote_transaction_receive_threads;
+    let vote_quic_server_config = QuicServerParams {
+        max_connections_per_peer: 1,
+        max_staked_connections: tpu_max_fwd_staked_connections.try_into().unwrap(),
```

Comment:
This should not be user-configurable to avoid people setting this too low accidentally. CRDS_UNIQUE_PUBKEY_CAPACITY is a good candidate for an upper bound here.